src/modules/places/content/placeController.js

```diff
@@ -314,6 +314,26 @@ PlacesController.prototype.do_cmd_places_singleClickExpand = function() {
     }
 }
 
+PlacesController.prototype.is_cmd_places_singleClickOpen_enabled = function() {
```

Comment:
I just copied these functions from clickToExpand :)
I'll use the SDK method in my addon, thanks.